src/index.js

```diff
@@ -283,6 +284,7 @@ class List extends Component {
 
 ReactDOM.render(<List />, document.querySelector('#mytable'));
 ReactDOM.render(<Filters />, document.querySelector('#filters'));
+ReactDOM.render(<Eligibility />, document.querySelector('#form-content'));
```

Comment:
This will not work as it is right now, because `document` refers to `public/index.html` from [this line](https://github.com/unicef/publicgoods-scripts/blob/312f3f6906ce8e0b1be01e32d259e95979cad0dd/index.js#L440). You are introducing a change in which you are modifying two different files: the registry where you have `#mytable` and `#filters` and the eligibility form where you have `#form-content`. If you don't tell it to distinguish between the two files, it can't. This fails right now because the registry does not have the `#form-content` container (because the eligibility page has it!)

Reply:
@lacabra Thank you for explaining what is going wrong. I am still somewhat confused about how to correct this error. What I have understood so far is that `public/index.html` contains the code for the the contents displayed on the registry page of the website. I don't understand what should `document` refer to so that `ReactDOM.render(<Eligibility />, document.querySelector('#form-content'));` displays the contents on the eligibility page of the website.